### PR TITLE
v0.3.1 Manual tests

### PR DESCRIPTION
**Pull request by 4ydan**
to #740

## Basics

- [ ] I added a line to [/doc/CHANGELOG.md](/doc/CHANGELOG.md)
- [x] The PR is rebased with current master.
- [x] Details of what you changed are in commit messages.
- [x] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.

## Checklist

- [ ] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed the introduction tour
- [ ] I wrote migrations in a way that they are compatible with already present data
- [ ] I fixed all affected decisions
- [ ] I added unit tests for my code
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
      (exceptions are documented)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] Examples are well chosen and understandable

### CODE REVIEW

**Comment by markus2330** [comment]
Thx for doing the tests :rocket:, you are a super-early :clock11:, not everything for the release is merged yet: see https://github.com/ElektraInitiative/PermaplanT/milestone/4

Note: In #714 I modified one test so that it succeeds now.

> I cant perform the guided tour tests.

Wait a bit until #735 is merged.

> How do I reenable the guided tour?

Either resetting the DB or using a different user.  #738 will hopefully provide a (partial) fix.

File: doc/tests/manual/reports/230730.md [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+
+## [Error Analysis](../README.md#report-header)
+
+No errors occured.
+
+## Closing remarks
+
+### How is the current state of the software?
+
+The software is in a fully working state with no issues. Maybe the usability of the grid scale on a map could be improved.
```
Comment (badnames) [inline doc/tests/manual/reports/230730.md:24]: How should the grid be changed in your opinion?

**Review by badnames — COMMENTED** [review]
I'm glad everything worked!

File: doc/tests/manual/reports/230730.md (hunk 1 above)
Comment (4ydan) [reply doc/tests/manual/reports/230730.md:24]: We fixed the problem in #744. The length indicator was in the way of the grid scale thingy.

**Comment by markus2330** [comment]
> I cant perform the guided tour tests.
> How do I reenable the guided tour?

@4ydan It is currently not possible, either you use local deployment, pr.permaplant.net or you create a new user.

For future: you could abort in the last step and tell that you want to continue next time.

**Comment by 4ydan** [comment]
I tested the tour on localhost.

**Pull request merged**